fiftyone/utils/qwen3_vl.py

```
@@ -0,0 +1,402 @@
+            return detections
+
+        try:
+            if json_str.startswith("```json"):
+                json_str = json_str[7:]
+            if json_str.startswith("```"):
+                json_str = json_str[3:]
+            if json_str.endswith("```"):
+                json_str = json_str[:-3]
+            json_str = json_str.strip()
+
+            if not (json_str.startswith("[") or json_str.startswith("{")):
```

Ruff (0.14.13)

Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)